### PR TITLE
F 19.1 - create review front end page (NOT READY)

### PR DESCRIPTION
**Pull request by bssammel**
have to resolve conflicts and test first

### CODE REVIEW

**Comment by bssammel** [comment]
Beware, fellow developers, for within this cursed pull request lies a malevolent force seeking to annihilate all in its path. 'f-19.1', the accursed branch, hungers for destruction, eradicating any semblance of version control in its wake.

Like a vile beast unleashed from its chains, it defies its patron deity 'dev', wreaking havoc without remorse. It seeks to kill the very essence of our codebase, leaving chaos and ruin in its wake.

Let us stand resolute against this foul curse. Fear not, for with the arrival of the new feature branch ✨'f-19.2'✨, peace shall be restored. This pull request, tainted by the darkness of 'f-19.1', shall be closed without mercy, its wicked influence banished forevermore. Trust in the arrival of 'f-19.2' to bring order back to our repository, restoring balance and harmony to our codebase once more as it heralds a new era with the create review front end. Embrace the light it brings, for it shall guide us towards a brighter future. 

@traceybee23 @craftycarmen just for a little giggle because everything hurts. 🧙

TLDR: this pull request will be abandoned because something is very wrong with the version control. Instead a new branch f-19.2 created  from dev will be created and the relevant changes from this PR will be  added to it.  I will close this PR when that branch is ready for merge to dev.

**Comment by bssammel** [comment]
Bye awful branch that I really really broke
